src/data/nat/bitwise.lean

```diff
@@ -207,4 +220,7 @@ begin
   exact lt_of_test_bit i (by simp [h, hi]) h (λ j hj, by simp [hi' _ hj])
 end
 
+lemma lt_lxor_cases {a b c : ℕ} (h : a < lxor b c) : lxor a c < b ∨ lxor a b < c :=
+(lxor_trichotomy h.ne).elim (λ h', (h.asymm h').elim) id
```

Comment:
I think this is `elim_left` or `elim_right` or something; you shouldn't need the `id`